src/components/qas/questions/qaFillableTable/FillableTableRangeInput.jsx

```diff
@@ -90,48 +99,58 @@ class FillableTableRangeInput extends React.PureComponent {
   };
 
   render = () => {
-    const { question, answer, activeId } = this.props;
-    const { answerable } = this.state;
+    const { question, answer } = this.props;
+    const { editing } = this.state;
     const { id, qaReview } = question;
-    const active = activeId === id;
     const answered = !isEmpty(answer);
     return (
       <>
         {qaReview && <StellarValueRange data={answer.data} />}
-        {!qaReview && (
-          <div className="table-cell-range-input">
-            <TextField
-              id={`range-input-${id}-min`}
-              type="text"
-              defaultValue={answered ? answer.data[0] : ''}
-              lineDirection="center"
-              onBlur={this.handleBlur}
-              onFocus={this.handleFocus}
-              onChange={this.handleMinChange}
-              disabled={!(answerable || answered || active)}
-              fullWidth={false}
-            />
-            <span>-</span>
-            <TextField
-              id={`range-input-${id}-max`}
-              type="text"
-              defaultValue={answered ? answer.data[1] : ''}
-              lineDirection="center"
-              onBlur={this.handleBlur}
-              onFocus={this.handleFocus}
-              onChange={this.handleMaxChange}
-              disabled={!(answerable || answered || active)}
-              fullWidth={false}
-            />
-          </div>
-        )}
+        <div className="table-cell-range-input">
+          {!qaReview && editing && (
+            <>
+              <TextField
+                id={`range-input-${id}-min`}
+                type="text"
+                defaultValue={answered ? answer.data[0] : ''}
+                lineDirection="center"
+                onBlur={this.handleBlur}
+                onFocus={this.handleFocus}
+                onChange={this.handleMinChange}
+                fullWidth={false}
+              />
+              <span> – </span>
+              <TextField
+                id={`range-input-${id}-max`}
+                type="text"
+                defaultValue={answered ? answer.data[1] : ''}
+                lineDirection="center"
+                onBlur={this.handleBlur}
+                onFocus={this.handleFocus}
+                onChange={this.handleMaxChange}
+                fullWidth={false}
+              />
+            </>
+          )}
+          {!qaReview && !editing && (
```

Comment:
added `editing` to the component state, if not currently editing then this button styled to appear as plaintext with an icon is displayed. On click the `handleEdit` method switches the state to editing mode and the text input is shown.